sklearn/svm/_classes.py

```diff
@@ -1541,29 +1546,32 @@ def __init__(
         )
 
     def fit(self, X, y=None, sample_weight=None, **params):
-        """Detects the soft boundary of the set of samples X.
+        """Detect the soft boundary of the set of samples X.
 
         Parameters
         ----------
         X : {array-like, sparse matrix} of shape (n_samples, n_features)
             Set of samples, where n_samples is the number of samples and
             n_features is the number of features.
 
+        y : Ignored
+            Not used, present for API consistency by convention.
+
         sample_weight : array-like of shape (n_samples,), default=None
             Per-sample weights. Rescale C per sample. Higher weights
             force the classifier to put more emphasis on these points.
 
-        y : Ignored
-            not used, present for API consistency by convention.
+        **params : dict
+            Additional fit parameters.
```

Comment:
To my knowledge, the `fit` function in the hierarchy of this class doesn't have any more parameters. Thus, `params` is not actually needed. Should we remove it?

Reply:
Yeah looks like it comes from BaseLibSVM. If you want you can check that passing any parameter actually raises an error, but it should be save to remove.

Reply:
Ok. When we are done with this PR, I will try to fix that in another PR.